sc/managers/configman/config_apply_handler.go

```diff
@@ -29,6 +32,12 @@ func (h *ConfigApplyHandler) Provision(ctx caddy.Context) error {
 	h.logger = ctx.Logger(h)
 	h.appLoader = ctx.App
 
+	store, err := ctx.App("configman")
+	if err != nil {
+		return err
+	}
+
+	h.store = store.(*ConfigMan).Connectors
```

Comment:
On further thought store the app directly in the handler. It will be a lot cleaner